docs/configuration.md

```diff
@@ -61,19 +61,19 @@ In plain english, the general idea is that each oomd config one or more
 RULESETs.  Each RULESET has a set of DETECTOR_GROUPs and a set of ACTIONs. Each
 DETECTOR_GROUP has a set of DETECTORs. Both DETECTORs and ACTIONs are PLUGIN
 types. That means _everything_ is a plugin in oomd. The rules on how a
-conforming config is evaulated at runtime are described in the next section.
+conforming config is evaluated at runtime are described in the next section.
 
 ### Notes
 
 * For `SILENCE_LOGS`, the currently supported log entities are
   * `engine`: oomd engine logs
   * `plugins`: logs written by plugins
 
-## Runtime evaulation rules
+## Runtime evaluation rules
 
 * Every plugin must return CONTINUE or STOP.
   * CONTINUE
-    * For DETECTORs, noop
+    * For DETECTORs, loop
```

Comment:
This is supposed to be `noop`, short for no-operation: https://en.wikipedia.org/wiki/NOP_(code)

Reply:
OK, I've updated following your request.